x/shared/helpers/service_configs.go

```diff
@@ -6,28 +6,93 @@ import (
 	sharedtypes "pocket/x/shared/types"
 )
 
-// AreValidAppServiceConfigs returns an error if the provided service configs are invalid
-// by wrapping the provided around with additional details
-func AreValidAppServiceConfigs(services []*sharedtypes.ApplicationServiceConfig) (string, bool) {
+// ValidateAppServiceConfigs returns an error if any of the application service configs are invalid
+func ValidateAppServiceConfigs(services []*sharedtypes.ApplicationServiceConfig) error {
 	if len(services) == 0 {
-		return fmt.Sprintf("no services configs provided for application: %v", services), false
+		return fmt.Errorf("no services configs provided for application: %v", services)
 	}
 	for _, serviceConfig := range services {
 		if serviceConfig == nil {
-			return fmt.Sprintf("serviceConfig cannot be nil: %v", services), false
+			return fmt.Errorf("serviceConfig cannot be nil: %v", services)
 		}
 		if serviceConfig.ServiceId == nil {
-			return fmt.Sprintf("serviceId cannot be nil: %v", serviceConfig), false
+			return fmt.Errorf("serviceId cannot be nil: %v", serviceConfig)
 		}
 		if serviceConfig.ServiceId.Id == "" {
-			return fmt.Sprintf("serviceId.Id cannot be empty: %v", serviceConfig), false
+			return fmt.Errorf("serviceId.Id cannot be empty: %v", serviceConfig)
 		}
 		if !IsValidServiceId(serviceConfig.ServiceId.Id) {
-			return fmt.Sprintf("invalid serviceId.Id: %v", serviceConfig), false
+			return fmt.Errorf("invalid serviceId.Id: %v", serviceConfig)
 		}
 		if !IsValidServiceName(serviceConfig.ServiceId.Name) {
-			return fmt.Sprintf("invalid serviceId.Name: %v", serviceConfig), false
+			return fmt.Errorf("invalid serviceId.Name: %v", serviceConfig)
 		}
 	}
-	return "", true
+	return nil
+}
+
+// ValidateSupplierService returns an error if any of the supplier service configs are invalid
+func ValidateSupplierService(services []*sharedtypes.SupplierServiceConfig) error {
```

Comment:
```suggestion
func ValidateSupplierServiceConfigs(services []*sharedtypes.SupplierServiceConfig) error {
```
NIT: Just to make it the same as the other not major tho